### DISCUSSION TITLE
VTKHDF output format

### DISCUSSION THREAD

**Discussion by Rahim-Habibi** [SeisSol/SeisSol#1189] [Q&A] [answered]
Hi all, 
I am working on small magnitude earthquake, so i need high resolution (high-order) output. 
To do so, i added "wavefieldvtkorder = 4 " and "numflux = 'rusanov'" in the input file and successfully have gotten the result. to visualise the output i open it in paraview, but once i am doing like filtering, Paraview starts to freeze. I think it can not handle it, right?
the size of the output is abou 1.2GB.
so, which software do you usually use for such high-order output. 
Thanks in advance, 
Rahim

**Comment by Thomas-Ulrich** [comment, marked as answer]
Hi,
I use  Paraview version 5.12.0, but don't think I tested with such a large file.
maybe start first with  "wavefieldvtkorder = 1"  (which would result in a much light file, to see if you can open it).
